### PR TITLE
Quick Start: track tap under "Menu"

### PR DESCRIPTION
**Pull request by leandroalonso**
This PR tracks when a user taps the Quick Start under "Menu".

### To test

1. Open the app
2. Start Quick Start for any site under "Menu"
3. Tap on the Quick Start cell
4. ✅ Make sure `quick_start_tapped` is fired

Please validate the event if everything looks fine.

## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Blog/QuickStartCell.swift [hunk 1]
```diff
@@ -14,7 +14,10 @@ import UIKit
 
         selectionStyle = .none
 
-        tourStateView.configure(blog: blog, sourceController: viewController)
+        let checklistTappedTracker: QuickStartChecklistTappedTracker = (event: .quickStartTapped, properties: [:])
+
+        tourStateView.configure(blog: blog, sourceController: viewController,
+                                checklistTappedTracker: checklistTappedTracker)
```
Comment (momo-ozawa) [inline WordPress/Classes/ViewRelated/Blog/QuickStartCell.swift]: [nit] Alignment
```suggestion
        tourStateView.configure(blog: blog, sourceController: viewController, checklistTappedTracker: checklistTappedTracker)
```

**Review by momo-ozawa — APPROVED** [review]
Works as described w 1 tiny nit. I validated the event ✅

**Comment by leandroalonso** [comment]
@mokagio FYI

**Comment by mokagio** [comment]
@leandroalonso   this has been bundled as part of [19.7 release candidate (19.7.0.3)](https://github.com/wordpress-mobile/WordPress-iOS/pull/18477).

Thanks for your work 🙌

**Pull request merged**